### PR TITLE
Setup and Navbar UI

### PR DESCRIPTION
**Pull request by atharvabhanage02**
### Folder Structure and Navbar
This PR contains **Mockbee files**  and  **Navbar Component**
**Files to Check :** 

- Navbar Component : src/components/Navbar/Navbar.jsx
- ProductPage : src/Pages/ProductListing/ProductPage.jsx

Here is the Live Preview Link :  https://deploy-preview-1--dainty-parfait-73cd3e.netlify.app/

### CODE REVIEW

File: src/Components/Navbar/navbar.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.navbar-wrapper{
+    position: fixed;
+    top: 0;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: center;
+    padding: 12px;
+    background-color: #8484e2;
```
Comment (sushil-mod) [inline src/Components/Navbar/navbar.css:10]: if possible give color variable

File: src/Components/Navbar/navbar.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
+.navbar-wrapper{
+    position: fixed;
+    top: 0;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: center;
```
Comment (sushil-mod) [inline src/Components/Navbar/navbar.css:8]: it is correct but you can try :- here display , flex-direction ,align-item properties are same as you used below where i tagged so you can create a class with these properties like flex-row-center

File: src/Components/Navbar/navbar.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    border: 2px solid black; 
+    border-radius: 1.56rem;
+}
+.wishlist-icon{
+    color: white;
+}
+.user-activity-details{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
```
Comment (sushil-mod) [inline src/Components/Navbar/navbar.css:38]: here display , flex-direction ,align-item are same

**Review by sushil-mod — APPROVED** [review]
your code is also fine but just a suggestion ,rest is good

**Review by JantuDeb — APPROVED** [review]
It's good but avoid adding large number files in one pull request.

File: src/Components/Navbar/navbar.css (hunk 1 above)
Comment (atharvabhanage02) [reply src/Components/Navbar/navbar.css:10]: Yes sure

**Pull request merged**